### ISSUE TITLE
Element.remove removes the element completely

### ISSUE THREAD

**Issue by russo-2025** [c-smile/sciter-js-sdk#149]
**Describe the bug**
Element.remove removes the element completely

**To Reproduce**
```
<html>
    <head>
        <style>
        </style>
        <script type="module">
            let el = document.createElement("div")
            el.remove();
            
            if(el) {
                console.log(!!el) //cast to bool
                console.log(el == null)
                console.log(el === null)

                el.style["width"] = "100px";
                el.style["height"] = "100px";
                el.style["background-color"] = "indianred";

                document.body.appendChild(el)
            }
            
        </script>
    </head>
    <body>
    </body>
</html>
```

**Expected behavior**
Element.remove should detach the element from the dom since the element is already detached, nothing will happen.
Element.remove должен отсоединить элемент от dom так, как элемент уже отсоединен, то ничего не произойдет.

**Screenshots**
![image](https://user-images.githubusercontent.com/28739948/126703122-87a0d5be-3ebf-4f24-9d28-896d47793449.png)

**Desktop (please complete the following information):
OS: win 10
Browser Microsoft Edge 91.0.864.64
Version 4.4.8.4

**Additional context**
it seems that there was a similar problem in Sciter.TIS
`Element.detach();`

**Comment by c-smile** [comment]
Sciter specific: disconnected element has `style`  sub-object undefined (reported as null).
Styles on element appear only when it is connected to particular place in the DOM.

So this works: 
```html
<html>
    <head>
        <style>
        </style>
        <script type="module">
            let el = document.createElement("div")
            el.remove();
            
            if(el) {
                
                document.body.appendChild(el);

                el.style["width"] = "100px";
                el.style["height"] = "100px";
                el.style["background-color"] = "indianred";

            }
            
        </script>
    </head>
    <body>
    </body>
</html>
```
